release/release_logs/fetch_release_logs.py

```diff
@@ -38,7 +38,7 @@
 from pybuildkite.buildkite import Buildkite
 
 BUILDKITE_ORGANIZATION = "ray-project"
-BUILDKITE_PIPELINE = "release-tests-branch"
+BUILDKITE_PIPELINE = "release"
```

Comment:
@can-anyscale For some reason the pipeline name has changed. The script only works if I update this name 😅 Let me know if we want to just update the script or if we should change make the pipeline name

Reply:
Ah yes thanks for fixing these! The new pipeline is perfect.